src/ngraph/op/concat.hpp

```diff
@@ -37,29 +37,35 @@ namespace ngraph
             ///
             /// \param args               The outputs producing the input tensors.
             /// \param axis The axis along which to concatenate the input tensors.
-            Concat(const OutputVector& args, size_t axis);
+            Concat(const OutputVector& args, int64_t axis);
 
             /// \brief Constructs a concatenation operation.
             ///
             /// \param args               The nodes producing the input tensors.
             /// \param axis The axis along which to concatenate the input tensors.
-            Concat(const NodeVector& args, size_t axis);
+            Concat(const NodeVector& args, int64_t axis);
 
             void validate_and_infer_types() override;
 
             virtual std::shared_ptr<Node>
                 copy_with_new_args(const NodeVector& new_args) const override;
 
             /// \return The concatenation axis.
-            size_t get_concatenation_axis() const { return get_axis(); }
-            void set_concatenation_axis(size_t concatenation_axis) { set_axis(concatenation_axis); }
+            int64_t get_concatenation_axis() const { return m_concat_axis; }
```

Comment:
@baojun-nervana @mozga-intel please change this to get_concat_axis() and I wouldn't expose a setter since this is an internal state variable, not to be manipulated directly by the op user.